### ISSUE TITLE
Add --pre to outdated command

### ISSUE THREAD

**Issue by mkevenaar** [chocolatey/choco#1804]
Hi,

I would like to be able to see all the outdated pre-releases of packages I have installed.

## Expected Behavior
Be able to run `choco outdated --pre` and see outdated packages, including pre-releases

## Current Behavior
The same as `choco outdated`

## Possible Solution
Add an option to allow pre-releases to be shown (e.g. using already known flag `--pre`)

## Steps to reproduce
1. Install the Chocolatey package using the install script provided in the documentation
2. run: `choco outdated --pre` 

## Expected response
```
Outdated Packages
 Output is package name | current version | available version | pinned?

chocolatey|0.10.13|0.10.14-beta-20190418|false
```

**Comment by ferventcoder** [comment]
The current behavior will see upgrades for installed prereleases, but this would cover wanting to see if there is a prerelease available for a stable package.

**Comment by ferventcoder** [comment]
This has been merged into stable at https://github.com/chocolatey/choco/commit/a1cda376ca0527733ae7458f3a33b4803869e3e8 and will be released with Chocolatey v0.10.14.
